src/composite.comp

```diff
@@ -76,7 +76,7 @@ vec4 sampleLayer(sampler2D layerSampler, uint layerIdx, vec2 uv) {
 
     if (coord.x < 0.0f       || coord.y < 0.0f ||
         coord.x >= texSize.x || coord.y >= texSize.y )
-        return vec4(0.0f, 0.0f, 0.0f, u_borderAlpha[layerIdx]);
+        return vec4(0.0f, 0.0f, 0.0f, float(u_borderMask & (1u << layerIdx)));
```

Comment:
This will end up as 1.0 for layer 0, 2.0 for layer 1, 3.0 for layer 2...
We probably always want 1.0.

Reply:
Fixed.